.gitignore

```diff
@@ -2,3 +2,4 @@
 *.iml
 /vendor/
 /build/
+node_modules/
```

Comment:
`js/` ?

Reply:
I'd like to keep the bundles committed so the app works right away after cloning.

Reply:
Maybe we should add a comment about this in the readme?
Saying it will complicate things etc etc.
Or we can set js to ignored, but still force add the compiled files?
That way you avoid people using the tutorial having their own compiled files all over the place? :)

Reply:
I actually thought about this.
A bot that force add and compile after each pull request merged.
That way it's far easier to work and rebase on pr, but you still have a working repo :thinking: